Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -179,8 +179,16 @@ private func pledgeCreditCardViewDataAndSelectedCard(
     return (card, isAvailableCardType, project.location.displayableName)
   }
 
-  // If there is no backing, simply select the first card in the list.
+  // If there is no backing, simply select the first card in the list when it is an available card type.
   guard let backing = project.personalization.backing else {
+    if let firstCardBrand = cards.first?.type?.rawValue {
```

Comment:
I think we could actually use the `data` that was computed in the line above, since we've already computed the first card's availability. This will save us from having to iterate through `availableCardTypes` to find the card brand again:

```
    guard let cardData = data.first, cardData.isEnabled else {
      return (data, nil)
    }

    return (data, cards.first)
```